website/README.md

```diff
@@ -84,32 +108,34 @@ git clone git@github.com:jordanpenard/brew-monitor.git .
 Install the required python libraries
 ```
 cd website
-pip3.7 install mod_wsgi
-pip3.7 install -r requirements.txt
+pip3.9 install mod_wsgi
+pip3.9 install -r requirements.txt
 ```
+Note: mod_wsgi is not in the `requirements.txt` because it requires apache to be installed. But maybe we should add it to ensure people have apache installed?
```

Comment:
I think you're right, `mod_wsgi` should be in `requirements.txt`, I'll let you make the change.